docs/install.md

```diff
@@ -3,26 +3,119 @@ id: install
 title: Install
 ---
 
-Install with `yarn`:
+First, install Prettier locally:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--npm-->
 
 ```bash
-yarn add prettier --dev --exact
-# or globally
-yarn global add prettier
+npm install --save-dev --save-exact prettier
 ```
 
-_We're using `yarn` but you can use `npm` if you like:_
+<!--yarn-->
 
 ```bash
-npm install --save-dev --save-exact prettier
-# or globally
-npm install --global prettier
+yarn add --dev --exact prettier
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+Then, create an empty config file to let editors and other tooling know you are using Prettier:
+
+```bash
+echo "{}" > .prettierrc.json
```

Comment:
How about 

```bash
echo {}> .prettierrc.json
```

Can't quote the string on windows

Reply:
Huh, I thought double quotes worked on Windows?

Reply:
Not with `echo`. `echo` sees and prints them.

Reply:
With

```bash
echo "{}" > .prettierrc.json
```

The content will be 

```
"{}"<SPACE>
```

Reply:
No repro on macOS Bash, Zsh, and Fish:

```shellsession
$ echo "{}" > foo.txt
$ cat foo.txt
{}
```